tests/Doctrine/Tests/DBAL/SQLParserUtilsTest.php

```diff
@@ -92,15 +92,6 @@ public function dataGetPlaceholderPositions()
         ];
     }
 
-    /**
-     * @dataProvider dataGetPlaceholderPositions
-     */
-    public function testGetPlaceholderPositions($query, $isPositional, $expectedParamPos)
```

Comment:
Should this be reworked in two tests, not deleted?

Reply:
Seems to be testing a deleted method.

Reply:
Yes, but the method was split apart into two other methods, not just deleted.

Reply:
Hmm but the test wasn't? That's weird. Ok.

Reply:
Ah, the methods are private.
How do you suggest to test them? Reflection?

Reply:
Split into two new tests and two data providers, using reflection - if you have better idea let me know. :)

Reply:
> Hmm but the test wasn't? That's weird. Ok.

I didn't notice it at that time.

> […] if you have better idea let me know. :)

It's good enough to me.